make/00_mod.mk

```diff
@@ -0,0 +1,48 @@
+# Copyright 2023 The cert-manager Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+repo_name := github.com/cert-manager/openshift-routes
+
+kind_cluster_name := openshift-routes
+kind_cluster_config := $(bin_dir)/scratch/kind_cluster.yaml
+
+build_names := manager
+
+go_manager_main_dir := ./internal/cmd
+go_manager_mod_dir := .
+go_manager_ldflags := -X $(repo_name)/pkg/internal/version.AppVersion=$(VERSION) -X $(repo_name)/pkg/internal/version.GitCommit=$(GITCOMMIT)
+oci_manager_base_image_flavor := static
+oci_manager_image_name := ghcr.io/cert-manager/cert-manager-openshift-routes
+oci_manager_image_tag := $(VERSION)
+oci_manager_image_name_development := cert-manager.local/cert-manager-openshift-routes
+
+deploy_name := openshift-routes
+deploy_namespace := cert-manager
+
+helm_chart_source_dir := deploy/charts/openshift-routes
+helm_chart_name := cert-manager-openshift-routes
+helm_chart_version := $(VERSION:v%=%)
```

Comment:
Not mandatory for this PR, but I think it's worth adding a warning in the README to explain that unlike other cert-manage charts, the openshift-routes chart uses versions without the `v` prefix. And explain why this weird decision was taken.